### ISSUE TITLE
Abstract: subcategories for games

### ISSUE THREAD

**Issue by Kreyren** [PhoenicisOrg/scripts#899]
Requesting to make subcategories for games to allow end-user to choose games e.i user wants to play some racing games? -> Phoenicis outputs all racing games that works, etc..

Reference: 
I like the idea to add application subcategories. There are two questions we need to answer in my opinion:
- how do application subcategories need to be modeled inside the script repository?
- how should they be shown visually? 

Regardless this should be discussed in its own issue. @Kreyren can you please create a new issue for this?

_Originally posted by @madoar in https://github.com/PhoenicisOrg/scripts/pull/877#issuecomment-473418524_

**Comment by plata** [comment]
Do you want distinct subcategories? Wouldn't it be better to use tags like Steam does?

**Comment by Kreyren** [comment]
@plata anything that makes searching in phoenicis easier,

expected if racing is selected -> output relevant supported apps

**Comment by plata** [comment]
Then I would really advice for tags over categories.

Just imagine something like this:
categories
- multiplayer
- shooter
Where do you put a multiplayer shooter?
With tags, just tag the multiplayer shooter with "multiplayer" and "shooter". Done.

**Comment by Kreyren** [comment]
> Where do you put a multiplayer shooter?

in multiplayer and shooter.

Tags are a thing already?

**Comment by plata** [comment]
https://store.steampowered.com/tag/

**Comment by Kreyren** [comment]
> https://store.steampowered.com/tag/

yep something alike might work assuming that tags are set by upstream

**Comment by madoar** [comment]
How would we tag the applications?
Add a new field to the `script.json` files?

**Comment by plata** [comment]
Or `application.json`, yes.

**Comment by madoar** [comment]
What about the visual parts? Any ideas how the tags should be visualizes and selected?

**Comment by plata** [comment]
Maybe like Steam does it?

**Comment by madoar** [comment]
So basically we add between the category selection and the filter checkboxes another selection component containing the tags for the selected category?

**Comment by plata** [comment]
Yes, something like that. We have to see how it works out with many tags.
